src/game/Tactical/Turn_Based_Input.cc

```diff
@@ -2729,6 +2747,25 @@ static void SetBurstMode(void)
 	if (sel != NULL) ChangeWeaponMode(sel);
 }
 
+static void GroupAutoReload()
+{
+	// modified from SwitchHeadGear
+
+	SOLDIERTYPE* const selectedSoldier = GetSelectedMan();
+	if (selectedSoldier == NULL) return;
+
+	FOR_EACH_IN_SQUAD(k, selectedSoldier->bAssignment)
```

Comment:
`FOR_EACH_IN_SQUAD` assumes `selectedSoldier->bAssignment` is lower than `NUMBER_OF_SQUADS`.
I would add an assert to ensure the assumption is true.

This code is probably based on `SwitchHeadGear`, which could also use an assert.

Reply:
Did something better, just grabbing the current squad, which is also globally tracked. The tactical map check also ensures it's one of the 20 normal slots.